js/const.js

```diff
@@ -42,6 +42,72 @@ define([
 				}
 			}]
 		};
+		
+		const getIncludedConceptsColumns = (sharedState, context, commonUtils, conceptSetService) => [
```

Comment:
I think it is the same as in this PR https://github.com/OHDSI/Atlas/pull/2199/files. Canyou combine them into one constant for use on both pages?